packages/documentation/src/content/docs/integration/requirements/peers.mdx

```diff
@@ -14,25 +15,47 @@ Refer to the [Rafiki Admin user guide](/admin/admin-user-guide#peers) for detail
 Whether you are using the Backend Admin API or the Rafiki Admin application, the underlying configurations and requirements remain the same. Choose the interface that best suits your individual workflow.
 :::
 
-## Prerequisites
+## Perform prerequisites
 
 :::note
 Peering is not required unless you want to participate in transactions with another ASE on the Interledger network. For foundational peering concepts, refer to the Peers section of [Interledger Concepts](/overview/concepts/interledger/#peers).
 :::
 
-Before adding a peer, you and the account servicing entity you intend to peer with must both:
+Before adding a peer, you, and the account servicing entity you intend to peer with must both:
```

Comment:
"Before adding a peer" is an introductory phrase. It's not a list of "a peer, you, and the account servicing entity". That comma after "you" comes out. Good flag though--maybe we can refine the Oxford Vale style a bit more. It's also a good reminder that not everything Vale suggests will necessarily be correct.